### PR TITLE
Drop closest from SelectorEngine

### PR DESCRIPTION
**Pull request by MartijnCuppens**
`SelectorEngine.closest()` just calls the native `.closest()`, which beats the point of having this function.

Not sure on which project board this should be added, @XhmikosR

### CODE REVIEW

File: js/src/button.js [hunk 1]
```diff
@@ -143,10 +139,7 @@ class Button {
 EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_TOGGLE_CARROT, event => {
   event.preventDefault()
 
-  let button = event.target
-  if (!button.classList.contains(CLASS_NAME_BUTTON)) {
```
Comment (XhmikosR) [inline js/src/button.js:147]: Is this intentional?

**Comment by XhmikosR** [comment]
Yeah, I guess it's a leftover from the IE removal PR.

File: js/src/button.js (hunk 1 above)
Comment (MartijnCuppens) [reply js/src/button.js:147]: Yes, `.closest()` returns the element itself if it contains the class (see https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)

**Comment by XhmikosR** [comment]
@Johann-S ping

**Review by Johann-S — APPROVED** [review]
LGTM 👍

**Pull request merged**